proposal/supports-general.md

```diff
@@ -0,0 +1,167 @@
+to allow raw Sass-script values to be used within the parentheses.
+
+The latter option is appealing at first glance. It requires fewer characters,
+and matches the parsing of `<supports-decl>`, which allows non-interpolated
+expressions on either side of the `":"`. However, it's only CSS-compatible as
+long as all the syntax CSS uses in those positions is also a valid SassScript
+expression. If not, Sass must continually update its syntax to support new CSS
+constructs.
+
+We've been burned by this in the past. Both this proposal and the [media ranges
+proposal][] would have been unnecessary (or at least much simpler) if the rules
+in questions had only allowed SassScript via interpolation in the first place.
+And it's likely that this syntax *will* evolve in ways that aren't
+SassScript-compatible, as with the `selector()` function [mentioned above][].
+
+[media ranges proposal]: ../accepted/media-ranges.md
+[mentioned above]: #background
+
+Given this, it's much safer to only allow SassScript within interpolation.
+
+#### Backwards Compatibility
+
+The `<general-enclosed>` syntax is very broad, so care is needed to ensure that
+adding support for it doesn't break existing Sass stylesheets—particularly those
+that are relying on SassScript in `<supports-decl>`. For example, strictly
+speaking `(foo + bar: baz)` is valid CSS that parses as `<general-enclosed>`,
+but Sass needs to continue to parse it as a declaration containing a SassScript
+expression on the left-hand side.
+
+The question then is how to balance backwards-compatibility with CSS
+compatibility. In most cases, CSS takes clear precedence, and we could manage
+that here by declaring that `<supports-decl>`s that begin with an
+`InterpolatedIdentifier` must immediately follow it with a `":"` and allowing
+all other syntax to be parsed as `<general-enclosed>`. However, this approach is
+not without its downsides. It would require a deprecation period, and it would
+parse `<general-enclosed>` in a number of cases (such as `(foo: )` or `(fo o:
+bar)`) that are much more likely to be typos.
+
+Fortunately, `<general-enclosed>` is explicitly specified for
+forwards-compatibility rather than having CSS semantics in and of itself, so we
+have more flexibility to limit compatibility with it in favor of Sass backwards
+compatibility. This allows us to limit the syntax we parse as
+`<general-enclosed>` to forbid a top-level `":"`, which ensures it's never
+ambiguous with a declaration and thus with raw SassScript.
+
+## Syntax
+
+### `SupportsCondition`
+
+<x><pre>
+**SupportsCondition**   ::= 'not' SupportsInParens
+&#32;                     | SupportsInParens ('and' SupportsInParens)*
+&#32;                     | SupportsInParens ('or' SupportsInParens)*
+**SupportsInParens**    ::= '(' (SupportsCondition | SupportsDeclaration | SupportsAnything) ')
+&#32;                     | SupportsFunction
+**SupportsDeclaration** ::= Expression² ':' Expression
```

Comment:
Style nit: could these footnotes be ordered?

Reply:
Oops, done.